app/Transformers/Outbound/Collections/Artwork.php

```diff
@@ -126,6 +126,13 @@ protected function getFields()
                     'type' => 'integer',
                 ],
             ],
+            'pageviews_shortterm' => [
```

Comment:
Bikeshedding moment on naming here, but this stuck out as a minor inconsistency – I saw that the field in the service was two words. I'm not sure if `shortterm` should be one word or two. Maybe `pageviews_recent`? I'm down to not worry about it.

Reply:
Yea, I wasn't sure what exactly to call this field. I'll change it to pageviews_recent, that's a good suggestion.